REGION_FORK_RECOMMENDATIONS.md

```diff
@@ -0,0 +1,193 @@
+
+1. This branch can contain files which don't exist in Covid Green's `current` branch.
+
+1. The only differences between `nys` and `current` should be files that do not exist in `current`.
+
+1. Changes in this branch are not allowed on files that exist in `current`.
+
+1. PRs can be accepted IFF they come from `nys-*` branches and do not contain changes to files in `current`.
+
+</td>
+</tr>
+
+<tr>
+<td>
+
+`nys-*` (or `penn-*`, `njs-*`, etc...)
+
+</td>
+<td>
+
+Branches containing changes to files only contained in `nys` branch.
+
+</td>
+<td>
+
+Forces regions to keep region specific changes distinct and separate from common project changes.
+
+</td>
+<td>
+
+1. These branches must start with `nys-` (or `penn-`, `njs-`, etc...).
+
+1. These branches must only contain updates to region specific files.
+
+1. Files in `current` may not be edited in these branches.
+
+1. These branches may only be PR'd into a region's main branch (`nys`, `penn`, `njs`, etc...).
+
+</td>
+</tr>
+<tr>
+<td>
+
+`cg-*`
+
+</td>
+<td>
+
+Branches containing changes to files in the `current` branch.
+
+</td>
+<td>
+
+1. Forces regions to keep common changes distinct and separate from region specific changes.
+
+1. Ensures that common changes are contributed back to Covid Green's `current` branch.
+
+1. Ensures that common changes are accepted into Covid Green before existing in a region's `current` branch.
+
+</td>
+<td>
+
+1. These branches must start with `cg-`.
+
+1. These branches must only contain updates to common project files.
+
+1. Region specific files cannot be edited in this branches.
+
+1. PRs may only be sent to Covid Green's `current` branch.
+
+</td>
+</tr>
+</table>
+
+## Staying Up to Date
+
+Periodic pulls from [covidgreen/covid-green-backend-api](https://github.com/covidgreen/covid-green-backend-api) will be necessary for a region specific fork to stay up to date with Covid Green. The following process can be used to reduce conflicts during these pulls.
+
+The examples assume the following configured remotes:
+
+```bash
+$> git remote -v
+green	git@github.com:covidgreen/covid-green-backend-api.git (fetch)
+green	git@github.com:covidgreen/covid-green-backend-api.git (push)
+vagabond	git@github.com:project-vagabond/covid-green-backend-api.git (fetch)
+vagabond	git@github.com:project-vagabond/covid-green-backend-api.git (push)
+  ## vagabond is the NYS fork of Covid Green
+
+```
+
+```bash
+# Switch to `current` branch 
+$> git co current
+
+# Fetch changes from Covid Green
+$> git fetch green
+
+# Merge Covid Green's `current` into the fork's `current`
+# Due to the rules above no merge conflicts should ever appear during this step.
+$> git merge green/current
+
+# NOTE: the fetch and merge can be done with a `git pull` if you'd rather single-step it
+
+# Rebase region specific branch off updated `current`
+$> git checkout nys # (or `git checkout penn`, `git checkout njs`, etc...)
+$> git rebase current
```

Comment:
I can't decide between `rebase` and `merge` here. Rebase feels a little cleaner but it'll result in the git history looking like all region specific changes came after all changes in Covid Green. Merge will keep the ordering but may potentially result in divergent histories across forks with different merge commits.

Reply:
I tend to prefer rebase for this kind of workflow. That said, if we're going to rebase then we should enforce that with no merge commits in the covidgreen-*/current branches.